src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable arrow-body-style */
```

Comment:
- [ ] Please refrain from using any `eslint-disable` in your code, as it defies the purpose of using linters in the first place.